LICENSES/README.md

```diff
@@ -0,0 +1,85 @@
+# systemd Project Licensing
+
+## Main License
+
+The systemd project uses a single-line reference to Unique License Identifiers
+in source files as defined by the Linux Foundation's SPDX project (https://spdx.org/).
+
+The current set of valid, predefined SPDX identifiers can be found on the SPDX
+License List at https://spdx.org/licenses/.
+
+The 'LICENSES/' directory contains all the licenses used by the sources included in
+the systemd project source tree.
+
+Unless otherwise noted, the systemd project sources are licensed under the terms
+and conditions of the **GNU Lesser General Public License v2.1 or later**.
+
+## Other Licenses
+
+The following exceptions apply:
+
+ * the following udev sources are licensed under **GPL-2.0-or-later**:
+   - src/udev/ata_id/*
+   - src/udev/cdrom_id/*
+   - src/udev/dmi_memory_id/*
+   - src/udev/mtd_probe/*
+   - src/udev/scsi_id/*
+   - src/udev/udevadm*
+   - src/udev/udev-builtin-blkid.c
+   - src/udev/udev-builtin.h
+   - src/udev/udev-builtin-input_id.c
+   - src/udev/udev-builtin-kmod.c
+   - src/udev/udev-builtin-path_id.c
+   - src/udev/udev-builtin-uaccess.c
+   - src/udev/udev-builtin-usb_id.c
+   - src/udev/udev-ctrl.h
+   - src/udev/udevd.c
+   - src/udev/udevd.h
+   - src/udev/udev-event.c
+   - src/udev/udev-event.h
+   - src/udev/udev-node.c
+   - src/udev/udev-node.h
+   - src/udev/udev-rules.c
+   - src/udev/udev-rules.h
+   - src/udev/udev-watch.c
+   - src/udev/udev-watch.h
+   - src/udev/v4l_id/*
+ * the header files contained in src/basic/linux/ and src/shared/linux/ are copied
+   verbatim from the Linux kernel source tree and are licensed under **GPL-2.0 WITH
+   Linux-syscall-note** and are used within the scope of the Linux-syscall-note
+   exception, and thus their license does not affect the rest of the source
+   code/binaries
+ * the src/shared/initreq.h header is licensed under original license,
+   **LGPL-2.0-or-later**.
+ * the src/shared/linux/bpf_insn.h header is copied from the Linux kernel
+   source tree and is licensed under either **BSD-2-Clause** or **GPL-2.0-only**,
+   and thus is included in the systemd build under the BSD-2-Clause license.
+ * The src/basic/linux/wireguard.h header is copied from the Linux kernel
+   source tree and is licensed under either **MIT** or **GPL-2.0 WITH Linux-syscall-note**,
+   and thus is included in the systemd build under the MIT license.
+ * the following sources are licensed under the **MIT** license:
+   - hwdb.d/parse_hwdb.py
+   - man/glib-event-glue.c
+   - src/basic/linux/batman_adv.h
+   - src/basic/sparse-endian.h
+   - tools/catalog-report.py
+ * the following sources are licensed under the **CC0-1.0** license:
+   - src/basic/siphash24.c
+   - src/basic/siphash24.h
+   - src/systemctl/systemd-sysv-install.SKELETON
+   - tools/check-includes.pl
+ * the following sources are under **Public Domain**:
+   - src/basic/MurmurHash2.c
+   - src/basic/MurmurHash2.h
+ * the following sources are under **Public Domain**:
```

Comment:
Why do you split `Public Domain` section into two?

Reply:
it's different texts (can't really call them licenses), so they have different SPDX identifiers

Reply:
Ah, indeed. But I do not think we should split the section in this document. But, not a big matter.

Reply:
added the ID in brackets to clarify

Reply:
Nice!

Reply:
> Ah, indeed. But I do not think we should split the section in this document. But, not a big matter.

The idea is to have a section for each of the SPDX identifiers used (apart from the default), to be extra-clear on what applies to which sources

Reply:
I'm getting a bit of DEP-5 vibes here :-)

Reply:
yeah, I tried to keep it human-readable as much as possible as the SPDX stuff is already the machine-friendly side, but it's  hard not to get repetitive :-)